### DISCUSSION TITLE
XBackBone Browser Extensions - for Mozilla based browsers e.g. LibreWolf or Chromium based, e.g. Dissenter

### DISCUSSION THREAD

**Discussion by yeehi** [SergiX44/XBackBone#477] [Ideas]
**Is your feature request related to a problem? Please describe.**
XBackBone might be more usable with a browser extension

**Describe the solution you'd like**
Publish extensions through e.g. F-Droid and mention they exist on the main XBackBone Website

**Additional context**
If one is making a screenshot, a lot of the time, a web-browser will be involved and it would be good to have some integration there.

**Comment by sergix44** [comment]
This is actually a good idea, maybe in the future i'll take a look for the v4 version
